pkgs/development/tools/tokio-console/default.nix

```diff
@@ -6,19 +6,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "tokio-console";
-  version = "0.1.7";
+  version = "0.1.9";
 
   src = fetchFromGitHub {
     owner = "tokio-rs";
     repo = "console";
     rev = "tokio-console-v${version}";
-    sha256 = "sha256-yTNLKpBkzzN0X73CjN/UXRGjAGOnCCgJa6A6loA6baM=";
+    sha256 = "sha256-zISgEhUmAfHErq4AelbnSwtKjtxYH//pbLUAlPKxQYk=";
```

Comment:
. . . and this to hash!

Reply:
Done.

Is `sha256` not supposed to be used anymore?
It's not deprecated and used in lots of modules.

Reply:
This is changing little by little.
Deprecation via warnings would trigger too much warnings - at least 25k files IIRC.

https://github.com/NixOS/nixpkgs/issues/245473